guides/moving-to-oss.md

```diff
@@ -0,0 +1,227 @@
+| **Supported IDEs**     | Web IDEs (code-server, Jupyter) + SSH-powered desktop IDEs (e.g. VS Code, JetBrains)                                                        | Same as Coder v1                                                                                                                    |
+| **Provisioner**        | Provisions workspaces on Kubernetes with hardcoded spec (pod + home volume)                                                                 | Provisions workspaces via [Terraform](https://terraform.io). Supports any resource (e.g. Windows VM, Kubernetes pod)                |
+| **Template**           | Optional YAML [configuration syntax](https://coder.com/docs/coder/latest/admin/templates) for workspaces. Managed by Coder admins or git/CI | [Terraform code](https://coder.com/docs/coder-oss/latest/templates) that defines workspace specs. Managed by Coder admins or git/CI |
+| **Image**              | Container image for workspace, contains dev tools and dependencies                                                                          | Container and VM image included in [the template](https://coder.com/docs/coder-oss/latest/templates)                                |
+| **Workspace options**  | CPU, RAM, GPU, disk size, image name, CVM (on/off), dotfiles                                                                                | Defined as variables in [the template](https://coder.com/docs/coder-oss/latest/templates)                                           |
+| **Deployment methods** | Kubernetes, Docker                                                                                                                          | Kubernetes, Docker, VM, or bare metal                                                                                               |
+| **Architecture**       | Control plane + PostgreSQL database + workspaces                                                                                            | Same as Coder v1                                                                                                                    |
+
+<small>Keep reading for an in-depth feature comparison. Also see the
+[Coder OSS documentation](https://coder.com/docs/coder-oss/)</small>
+
+## Migration Strategy
+
+A seperate deployment is necessary to run Coder OSS. A direct upgrade via Helm
+is not possible since Coder OSS introduces new concepts (e.g. templates,
+provisioners) and other features are still being developed (e.g. audit log,
+organization support).
+
+Short term, we recommend keeping your Coder v1 deployment and inviting a pilot
+group to your Coder OSS deployment to reproduce their workflows and try new
+features (e.g. Windows support, dynamic secrets, faster builds).
+
+### Feature list key
+
+Please comment on any issues in the
```

Comment:
Suggestion - feel free to incorporate as needed. 

each of the following features have open issues on coder/coder - if they're a priority for your team, please chime in on the github issue to help our team prioritize our communities needs first.

Reply:
thanks, adding!